### ISSUE TITLE
Bind commands?

### ISSUE THREAD

**Issue by vspinu** [nicferrier/emacs-noflet#16]
Is there a way to re-bind commands. Something like?

```
(noflet ((test1 () (interactive) (insert "test1")))
  (call-interactively 'test1))
```

**Comment by DarwinAwardWinner** [comment]
So, I just implemented this: http://stackoverflow.com/a/29668310/125921

@nicferrier, do you have any interest in incorporating this into noflet?

**Comment by DarwinAwardWinner** [comment]
Ok, now it's a pull request.

**Comment by vspinu** [comment]
@DarwinAwardWinner thanks! Is it completely transparent to the user and one `(interactive)` inside a function would be enough?

**Comment by DarwinAwardWinner** [comment]
Yep, it just works the way you expect it to. And it works on your example.

**Comment by DarwinAwardWinner** [comment]
Actually, this code is broken in a few corner cases. Give me some time to fix it up before merging.
